browser/brave_ads/ads_tooltips_delegate_impl.cc

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_ads/ads_tooltips_delegate_impl.h"
+
+#include <string>
+#include <utility>
+
+#include "chrome/browser/profiles/profile.h"
```

Comment:
you're missing a dep for `//chrome/browser/profiles:profile`

Reply:
Thanks, added.